kubernetes/zeus/zeus/nfs-pv.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nfs
+spec:
+  capacity:
+    storage: 500Gi
+  accessModes:
+    - ReadWriteMany
+  storageClassName: standard
+  nfs:
+    path: /
+    server: 10.43.247.96
```

Comment:
Not sure if this is brittle or not. Just pointing it out. It's also publishing an internal IP address for the NFS host if there was some RCE. Probably not practically a concern.

Reply:
I couldn't find a way to make this easily discoverable unfortunately.